Helper/Elasticsearch/ClientRequestRuntime.php

```diff
@@ -77,4 +80,37 @@ public function data(string $input)
 
         return ($total > 1) ? false : null;
     }
+
+    /**
+     * @param string $input
+     *
+     * @return array|false|null false when multiple results
+     */
+    public function get(string $input)
```

Comment:
This is a new function, so be specific over output type  ?array
But, I think changing to emsch_get should be done with support of an object as output, an no longer the array as discussed. Only giving [_source] as output is also cutting away valuable information about the object (content type) for later use in code!

Reply:
Yes indeed, It can be very helpful to have in object in the future! And the _id is also import information and not available in the source.

Reply:
See 68c481f3d1b4f029a73b6f277e90c7b577204fed